tests/client/test_dataset_version_export.py

```diff
@@ -0,0 +1,128 @@
+            datasets=[resources_pb2.Dataset(id=dataset_id)],
+        ),
+        metadata=metadata(),
+    )
+    raise_on_failure(post_datasets_response)
+
+    try:
+        post_inputs_response = stub.PostInputs(
+            service_pb2.PostInputsRequest(
+                inputs=[
+                    resources_pb2.Input(
+                        data=resources_pb2.Data(
+                            image=resources_pb2.Image(
+                                url=DOG_IMAGE_URL,
+                                allow_duplicate_url=True,
+                            ),
+                        ),
+                        dataset_ids=[dataset_id],
+                    ),
+                    resources_pb2.Input(
+                        data=resources_pb2.Data(
+                            image=resources_pb2.Image(
+                                url=TRUCK_IMAGE_URL,
+                                allow_duplicate_url=True,
+                            ),
+                        ),
+                        dataset_ids=[dataset_id],
+                    ),
+                ],
+            ),
+            metadata=metadata(),
+        )
+        raise_on_failure(post_inputs_response)
+
+        input_ids = [input.id for input in post_inputs_response.inputs]
+        wait_for_inputs_upload(stub, metadata(), input_ids)
+
+        dataset_version_id = dataset_id  # Reuse dataset external ID for version.
+        post_dataset_versions_response = stub.PostDatasetVersions(
+            service_pb2.PostDatasetVersionsRequest(
+                dataset_id=dataset_id,
+                dataset_versions=[resources_pb2.DatasetVersion(id=dataset_version_id)],
+            ),
+            metadata=metadata(),
+        )
+        raise_on_failure(post_dataset_versions_response)
+
+        wait_for_dataset_version_ready(stub, metadata(), dataset_id, dataset_version_id)
+
+        put_dataset_version_exports_response = stub.PutDatasetVersionExports(
+            service_pb2.PutDatasetVersionExportsRequest(
+                dataset_id=dataset_id,
+                dataset_version_id=dataset_version_id,
+                exports=[
+                    resources_pb2.DatasetVersionExport(
+                        format=resources_pb2.CLARIFAI_DATA_PROTOBUF,
+                    ),
+                ],
+            ),
+            metadata=metadata(),
+        )
+        raise_on_failure(put_dataset_version_exports_response)
+
+        wait_for_dataset_version_export_success(stub, metadata(), dataset_id, dataset_version_id)
+
+        get_dataset_version_response = stub.GetDatasetVersion(
+            service_pb2.GetDatasetVersionRequest(
+                dataset_id=dataset_id,
+                dataset_version_id=dataset_version_id,
+            ),
+            metadata=metadata(),
+        )
+        raise_on_failure(get_dataset_version_response)
+
+        export = get_dataset_version_response.dataset_version.export_info.clarifai_data_protobuf
```

Comment:
`export_info` should only be undefined in case this test was called on a dataset version without calling `PutDatasetVersionExports` first, which makes it a test error. But even if this is a server error, just throwing an `AttributeError` for `NoneType` should be an understandable exception of what happened.